ch1/study2.html

```diff
@@ -3,19 +3,41 @@
   <head>
     <meta charset="utf-8" />
     <title>This is TITLE!!</title>
+    <link rel="stylesheet" href="./study.css" />
   </head>
   <body>
-    <h1>This is h1</h1>
-    <!--This is comment and h1 is very important!-->
-    <a href=https://www.naver.com>This is hyperLink feat. a tag</a>
-    <h2>This is h2</h2>
-    <fieldset>
-      <legend>Legend tag is name</legend>
-      <input type="text" />
-      <input type="checkbox" />
-      <input type="radio" />
-      <!--Various input type.-->
-      <button>button</button>
-    </fieldset>
+    <header> <!-- 헤더 영역을 선택할 때, header태그로 묶어줌. -->
+      <div id="main-item"> <!-- 영역을 선택할 때, div태그로 묶어줌. -->
+        <h1>This is h1</h1>
+        <!-- This is comment and h1 is very important! -->
+        <a href=https://www.naver.com>This is hyperLink feat. a tag</a>
+        <h2>This is h2</h2>
+        <fieldset>
+          <legend>Legend tag is name</legend>
+          <input type="text" />
+          <input type="checkbox" />
+          <input type="radio" />
+          <!--Various input type.-->
+          <button>button</button>
+        </fieldset>
+      </div>
+    </header>
+    <div>
+      <ul> <!-- 정렬되지 않은 리스트 -->
+        <li>This is item</li>
+        <li>This is item2</li>
+        <li>This is item3</li>
+        <li>This is item4</li>
+      </ul>
+      <img src="https://user-images.githubusercontent.com/1487073/58344409-70473b80-7e0a-11e9-8570-b2efc6f8fa44.png" alt="VSCode Image" width="640" height="380"/>
```

Comment:
```suggestion
      <img src="https://user-images.githubusercontent.com/1487073/58344409-70473b80-7e0a-11e9-8570-b2efc6f8fa44.png" alt="VSCode Image" width="640" height="380" />
```

* 닫는 태그는 띄어 씁니다.
* 이미지 태그에 `alt` 붙여주는거 좋습니다. 항상 있어야 합니다.